wrapper/src/main/resources/lang/de_de.json

```diff
@@ -0,0 +1,84 @@
+  "utils.utils.gicMessage.serverUsedMemory": "\n\nVom Server genutzter Speicher:\n%s MB / %s MB",
+
+  "utils.utils.gscMessage.scoreboard": "Scoreboard",
+  "utils.utils.gscMessage.noResult": "\nkein Ergebnis",
+
+  "utils.utils.ubCommands.help": "Frage eine Liste aller Commands an",
+  "utils.utils.ubCommands.info": "Frage den aktuellen Server Status an",
+  "utils.utils.ubCommands.stats": "Frage das Scoreboard für eine Statistik an",
+  "utils.utils.ubCommands.stats.type": "Statistik Typ",
+  "utils.utils.ubCommands.stats.name": "Statistik Name",
+  "utils.utils.ubCommands.update": "Überprüfe ob neue Updates verfügbar sind",
+  "utils.utils.ubCommands.whitelist": "Füge einen Spieler zur Server whitelist hinzu",
+  "utils.utils.ubCommands.whitelist.player": "Spieler Name",
+  "utils.utils.ubCommands.console": "Führe einen Befehl auf der Server Konsole aus",
+  "utils.utils.ubCommands.console.command": "Befehl, welcher ausgeführt werden soll",
+  "utils.utils.ubCommands.log": "Frage einen Server-Log an",
+  "utils.utils.ubCommands.log.file": "Dateiname",
+  "utils.utils.ubCommands.reload": "Lade MC-Discord-Chat Konfigurations-Datei neu",
+  "utils.utils.ubCommands.stop": "Stoppe den Server",
+
+  "message.unformattedResponseMessage": "    ┌──── <%name%> %message%",
+  "message.unformattedChatMessage": "[%server%] <%name%> %message%",
+  "message.unformattedOtherMessage": "[%server%] %message%",
+  "message.unformattedCommandNotice": "%name% hat %command% ausgeführt!",
+
+  "message.formattedResponseMessage": "[{\"text\":\"    ┌──── \",\"bold\":true,\"color\":\"dark_gray\"},{\"text\":\"<%name%> \",\"bold\":false,\"color\":\"%roleColor%\"},{\"text\":\"%message%\",\"bold\":false,\"color\":\"dark_gray\"}]",
+  "message.formattedChatMessage": "[{\"text\":\"[%server%] \",\"bold\":true,\"color\":\"blue\"},{\"text\":\"<%name%> \",\"bold\":false,\"color\":\"%roleColor%\"},{\"text\":\"%message%\",\"bold\":false,\"color\":\"gray\"}]",
+  "message.formattedOtherMessage": "[{\"text\":\"[%server%] \",\"bold\":true,\"color\":\"blue\"},{\"text\":\"%message%\",\"bold\":false,\"color\":\"gray\"}]",
+  "message.formattedCommandNotice": "[{\"text\":\"%name% \",\"bold\":false,\"color\":\"%roleColor%\"},{\"text\":\"executed %command% command!\",\"bold\":false,\"color\":\"gray\"}]",
+
+  "message.messageWithoutWebhook": "<%name%> %message%",
+  "message.messageWithoutWebhookForMultiServer": "[%server%] <%name%> %message%",
+
+  "message.serverStarted": "**Server gestartet!**",
+  "message.serverStopped": "**Server gestoppt!**",
+
+  "message.joinServer": "**%playerName% ist dem Spiel beigetreten**",
+  "message.leftServer": "**%playerName% hat das Spiel verlassen**",
+
+  "message.deathMessage": "**%deathMessage%**",
+
+  "message.advancementTask": "**%playerName% hat den Fortschritt [%advancement%] gemacht**\n*%description%*",
+  "message.advancementChallenge": "**%playerName% hat die challenge [%advancement%] abgeschlossen**\n*%description%*",
```

Comment:
Hello. Could you confirm whether "challenge" was forgotten to be translated?

Reply:
"Challenge" is also okay in German, although it has to be capitalised.

Reply:
Ok, thanks for the explanation!